### PR TITLE
Update tags for CoreInteractiveGraph plugins

### PR DESCRIPTION
**Pull request by sol695510**
### Description of the Change
Updated tags for CoreInteractiveGraph plugins.

### Alternate Designs
N/A

### Why Should This Be In Core?
N/A

### Benefits
N/A

### Possible Drawbacks
N/A

### Verification Process
N/A

### Applicable Issues
#1126

### CODE REVIEW

File: CoreInteractiveGraph/src/au/gov/asd/tac/constellation/graph/interaction/plugins/io/AutosaveGraphPlugin.java [hunk 1]
```diff
@@ -56,8 +56,8 @@
  * @author algol
  */
 @ServiceProvider(service = Plugin.class)
-@PluginInfo(pluginType = PluginType.NONE, tags = {"LOW LEVEL"})
 @Messages("AutosaveGraphPlugin=Autosave Graph")
+@PluginInfo(pluginType = PluginType.UPDATE, tags = {"MODIFY"})
```
Comment (arcturus2) [inline CoreInteractiveGraph/src/au/gov/asd/tac/constellation/graph/interaction/plugins/io/AutosaveGraphPlugin.java:60]: I was thinking `EXPORT` for the auto save as its creating a file outside of Constellation.

File: CoreInteractiveGraph/src/au/gov/asd/tac/constellation/graph/interaction/plugins/io/SaveGraphPlugin.java [hunk 2]
```diff
@@ -44,6 +46,7 @@
  */
 @ServiceProvider(service = Plugin.class)
 @NbBundle.Messages("SaveGraphPlugin=Save Graph")
+@PluginInfo(pluginType = PluginType.UPDATE, tags = {"MODIFY"})
```
Comment (arcturus2) [inline CoreInteractiveGraph/src/au/gov/asd/tac/constellation/graph/interaction/plugins/io/SaveGraphPlugin.java:49]: Similarly I was thinking `EXPORT` as its saving to a file.

**Review by arcturus2 — APPROVED** [review]

**Comment by arcturus2** [comment]
@sol695510 @antares1470 @aldebaran30701 in anticipation of creating a RC for next week I will merge this as is without your review. Any issues just reply and let me know.

File: CoreInteractiveGraph/src/au/gov/asd/tac/constellation/graph/interaction/plugins/clipboard/CopyToClipboardAction.java [hunk 3]
```diff
@@ -54,13 +56,24 @@ public void actionPerformed(final ActionEvent e) {
...
+        public String getName() {
+            return "Copy To Clipboard Action: Copy To Clipboard";
+        }
 
-            }
-        }).executeLater(graph);
+        @Override
+        protected void execute(final PluginGraphs graphs, final PluginInteraction interaction, final PluginParameters parameters) throws InterruptedException, PluginException {
+            final Graph graph = graphs.getGraph();
+            PluginExecution.withPlugin(InteractiveGraphPluginRegistry.COPY).executeNow(graph);
```
Comment (antares1470) [inline CoreInteractiveGraph/src/au/gov/asd/tac/constellation/graph/interaction/plugins/clipboard/CopyToClipboardAction.java:76]: This is just more of an aside but it almost feels like this plugin is unnecessary. All it's doing is executing another plugin on a graph and it appears there is a graph gotten earlier on in the action (the one from the context). Is there something wrong with that graph that means we need to grab the graph differently here? @arcturus2

File: CoreInteractiveGraph/src/au/gov/asd/tac/constellation/graph/interaction/plugins/io/AutosaveGraphPlugin.java (hunk 1 above)
Comment (antares1470) [reply CoreInteractiveGraph/src/au/gov/asd/tac/constellation/graph/interaction/plugins/io/AutosaveGraphPlugin.java:60]: Agree

File: CoreInteractiveGraph/src/au/gov/asd/tac/constellation/graph/interaction/plugins/io/SaveGraphPlugin.java (hunk 2 above)
Comment (antares1470) [reply CoreInteractiveGraph/src/au/gov/asd/tac/constellation/graph/interaction/plugins/io/SaveGraphPlugin.java:49]: Agree

Comment (sol695510) [reply CoreInteractiveGraph/src/au/gov/asd/tac/constellation/graph/interaction/plugins/io/SaveGraphPlugin.java:49]: #1259

File: CoreInteractiveGraph/src/au/gov/asd/tac/constellation/graph/interaction/plugins/io/AutosaveGraphPlugin.java (hunk 1 above)
Comment (sol695510) [reply CoreInteractiveGraph/src/au/gov/asd/tac/constellation/graph/interaction/plugins/io/AutosaveGraphPlugin.java:60]: #1259

File: CoreInteractiveGraph/src/au/gov/asd/tac/constellation/graph/interaction/plugins/clipboard/CopyToClipboardAction.java (hunk 3 above)
Comment (arcturus2) [reply CoreInteractiveGraph/src/au/gov/asd/tac/constellation/graph/interaction/plugins/clipboard/CopyToClipboardAction.java:76]: No there isn't @antares1470 and we could actually replace this to extend `SimplePluginAction`. I will have a look at doing this after the RC2 is done.

**Pull request merged**